### ISSUE TITLE
Implement SUMMARY display for `top`

### ISSUE THREAD

**Issue by Krysztal112233** [uutils/procps#207]
Currently our implementation output:

```
TODO

```

But the GNU's implementation:

```
top - 15:51:38 up 9 min,  1 user,  load average: 1.45, 1.09, 0.61
Tasks: 496 total,   1 running, 495 sleeping,   0 stopped,   0 zombie
%Cpu(s):  3.5 us,  1.7 sy,  0.0 ni, 94.8 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st 
MiB Mem :  31506.4 total,  20616.9 free,   5700.0 used,   5692.4 buff/cache     
MiB Swap:      0.0 total,      0.0 free,      0.0 used.  25806.5 avail Mem 

```

**Comment by Bluemangoo** [comment]
Hi, I'd like to work on this.

I found another crate `systemstat`, which partially overlaps with `sysinfo`, but provides something more like uptime and cpu time. Can we use it?

**Comment by cakebaker** [comment]
@Bluemangoo sure, go ahead :) It might make sense to split it into multiple PRs, similar to the documentation in the manpage.

There is also someone working on a TUI in https://github.com/uutils/procps/pull/292, but I think this issue can be fixed independently.

**Comment by r3yc0n1c** [comment]
yes, I was working on this... didn't get much time lately... if you plan to use the multi-threaded background to refresh just like Krysztal112233 suggested in the PR, please refer to this discussion: https://github.com/uutils/procps/pull/292#discussion_r1899896479 and It would be great if you could implement it in a way that allows me to call and render these from the `tui` mod.
